KarhooUISDK/Screens/CheckoutScreen/NewCheckout/Workers/KarhooNewCheckoutPaymentAndBookingWorker.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  KarhooNewCheckoutPaymentWorker.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 12/01/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+import UIKit
+import SwiftUI
+import Combine
+
+enum CheckoutBookingError {
+    case noUser
+    case cardAuthenticationFailure(message: String?)
```

Comment:
im Booking+MVPR.swift file we have: 
`public enum BookingScreenResult {
    case tripAllocated(tripInfo: TripInfo)
    case prebookConfirmed(tripInfo: TripInfo)
    case bookingFailed(error: KarhooError)
}`
Why do we need to introduce new enum for Booking errors?

Reply:
We can use it (or at least try to), but it'd be in next ticket, once the flow is ready.

Reply:
We can use it (or at least try to), but it'd be in a next ticket, once the flow is ready.

Reply:
We can use it (or at least try to), but it'd be in a next ticket, once the flow is ready.